cid/common.py

```diff
@@ -385,7 +385,8 @@ def _deploy(self, dashboard_id: str=None, recursive=True, update=False, **kwargs
 
                 if not matching_datasets:
                     logger.warning(f'Dataset {dataset_name} is not found')
-                    raise CidCritical(f'Dataset "{dataset_name}" ({ds.id}) is missing required fields. {(unmatched)}')
+                    raise CidCritical(f'Failed to find a Dataset "{dataset_name}" with required fields. Please check existing DataSets. '
```

Comment:
Add a check if it's executed in CFN Lambda or in console and use different error messages for both cases

Reply:
done